integration-tests/src/test/java/oracle/weblogic/kubernetes/ItRetryImprovements.java

```diff
@@ -106,13 +122,18 @@ public static void initAll(@Namespaces(2) List<String> namespaces) {
 
     // install and verify operator
     installAndVerifyOperator(opNamespace, opServiceAccount, false, 0, domainNamespace);
+
+    // get operator pod name
+    operatorPodName = assertDoesNotThrow(() -> getOperatorPodName(OPERATOR_RELEASE_NAME, opNamespace));
+    logger.info("operator pod name is: {0}", operatorPodName);
   }
 
   // This method is needed in this test class to delete uncompleted domain to restore the env
   @AfterEach
   public void tearDown() {
     try {
       deleteSecret(wlSecretName, domainNamespace);
+      deleteClusterCustomResource(clusterName, domainNamespace);
 
       Callable<Boolean> domain = domainExists(domainUid, DOMAIN_VERSION, domainNamespace);
       if (domain.call().booleanValue()) {
```

Comment:
this will be same answer for your comments for configMapExist.call().booleanValue(). Sorry I can't. :( because it returns Callable but if requires boolean

Reply:
public static Callable<Boolean> domainExists(String domainUid, String domainVersion, String namespace) {
    return () -> Domain.doesDomainExist(domainUid, domainVersion, namespace);
  }

Are we looking at a diff method?

Reply:
The test uses import static oracle.weblogic.kubernetes.assertions.TestAssertions.domainExists;, not Domain.java